docs/python/reference/dbos-class.md

```diff
@@ -32,6 +32,7 @@ DBOS.launch()
 
 Launch DBOS, initializing database connections and starting scheduled workflows.
 Should be called after all decorators run.
+**You should not call a DBOS function until after DBOS is launched.**
```

Comment:
This information is quite important. I wonder if we want to highlight it in an `info` box, also in the learn python page?

Reply:
The error messages you get when you don't are unambiguous, and it isn't an issue at all with FastAPI, so I'm not too concerned.